services.html

```diff
@@ -148,7 +100,9 @@ <h3 class="post-heading">Latest Post</h3>
   </footer>
   <script type="module" src="resources/js/utils.js"></script>
   <script type="module" src="resources/js/common.js"></script>
-</body>
-</html>
+  <script type="module" src="resources/js/services.js"></script>
+  <script type="module" src="resources/js/table.js"></script>
```

Comment:
Should we not link _table.js_ before _services.js/home.js_

Reply:
Done